src/rules/poisson/out.jl

```diff
@@ -0,0 +1,5 @@
+export rule
+
+@rule Poisson(:out, Marginalisation) (m_l::PointMass,) = Poisson(mean(m_l))
+
+@rule Poisson(:out, Marginalisation) (q_l::GammaDistributionsFamily,) = Poisson((shape(q_l) - 1 // 2) / rate(q_l))
```

Comment:
I end up with a digamma function here: `lambda = exp(digamma(alpha))/beta`. Also saw that there are some approximations to this term: https://en.wikipedia.org/wiki/Digamma_function#Computation_and_approximation. The question is whether we always want to use this approximation.